Formula/xplanet.rb

```diff
@@ -38,6 +38,11 @@ class Xplanet < Formula
   depends_on :x11 => :optional
 
   def install
+    # This is a simple fix to a bug in Xplanet 1.3.1 (-num_times=2 doesn't work):
+    # http://xplanet.sourceforge.net/FUDforum2/index.php?t=msg&th=671&start=0&S=07920715eb9a74361428ce957b99ae66
+    # This block should be removed with the next release of Xplanet
+    inreplace "src/libdisplay/DisplayOutput.cpp", "if (stopIndex > 1)", "if (stopIndex > 0)"
```

Comment:
I think we'll probably want to use a `patch` block here instead, as that forum link suggests that a patch is available in the SVN repo.